monosi/anomalies/__init__.py

```diff
@@ -41,29 +38,42 @@ def _z_scores(cls, stats: List[MetricStat], filter_none=True):
 
         if (std_dev == 0): return []
 
-        for stat in stats:
-            stat.std_dev = std_dev
+        z_scores = []
+        for value in values:
             try:
-                if stat.value:
-                    stat.z_score = round(((stat.value - mean) / std_dev), 2) 
+                z_score = round(((value - mean) / std_dev), 2) 
```

Comment:
Divide by 0 *shouldn't* happen here because we do `if (std_dev == 0): return []`, right?

Reply:
implicitly via catching the exception, yes